src/gateway/kong-enterprise/index.md

```diff
@@ -4,89 +4,97 @@ badge: enterprise
 content_type: explanation
 ---
 
-In addition to the {{site.base_gateway}} open source, free, and plus features, users with an Enterprise plan also have access to Enterprise-only features. These features enhance the security, monitoring, and management capabilities of {{site.base_gateway}}.
+Kong Enterprise is the scalable, secure, and flexible API management solution that extends Kong Gateway, the fastest, most adopted API gateway, with enterprise plugins, developer portal, analytics, advanced security features, GUI's, and 24/7 support. It is the only solution that helps you accelerate your cloud journey by managing, securing, and monitoring connections between applications across hybrid and multi-cloud architectures, to help you scale faster and boost developer productivity. 
```

Comment:
There's a lot of marketing-speak here. 
"fastest, most adopted API gateway." - can we guarantee that? Is that our official promise?  
"It is the only solution that helps you accelerate...etc" - is it the only solution? Do we know that?